### ISSUE TITLE
Refused to execute inline script

### ISSUE THREAD

**Issue by metapodcod** [BrowserSync/browser-sync#1923]
### Issue details

_Please provide issue details here_.

- Even though I didn't make any changes to my code, when I woke up in the morning I ran my "gulp" project and got such a strange error

`Refused to execute inline script because it violates the following Content Security Policy directive: "default-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-ACotEtBlkqjCUAsddlA/3p2h7Q0iHuDXxk577uNsXwA='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.`

- the seventh line is said to have an error, the content of the seventh line is as follows

```
<script id="__bs_script__">//<![CDATA[
    document.write("<script async src='/browser-sync/browser-sync-client.js?v=2.27.7'><\/script>".replace("HOST", location.hostname));
//]]></script>
```

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [ 2.27.7    ]
- Node        [  14.18.2  ]
- Npm         [  8.3.0  ]

### Affected platforms

- [ x ] linux- Windows10 WSL

### Browsersync use-case

- [ x ] Gulp

**Comment by curtisbarnard** [comment]
I'm getting this same response on an 11ty project that I'm working on...

**Comment by eredwan** [comment]
I am having the same problem trying to run in Angular13. This wasnt happening before I upgraded running on Angular 5. Does anyone have any ideas on how to fix this? Extremely frustrating

**Comment by chen-xin-08** [comment]
我也遇到了同样的问题，不知如何解决

**Comment by shakyShane** [comment]
fixed in `browser-sync@2.28.0` :)
